src/sonic-ctrmgrd/ctrmgr/ctrmgrd.py

```diff
@@ -172,11 +176,11 @@ def register_db(self, db_name):
             self.db_connectors[db_name] = swsscommon.DBConnector(db_name, 0)
 
 
-    def register_timer(self, ts, handler):
+    def register_timer(self, ts, handler, args=()):
```

Comment:
>args=() [](http://example.com/codeflow?start=42&length=7)

Even it is allowed to use empty tuple (which is immutable) as default value, high suggest using `None` as default value.